<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Doubango Project</author>
 <class>doubsClass</class>
 <widget class="QMainWindow" name="doubsClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>doubs</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>65</y>
      <width>320</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOnline"/>
    <addaction name="actionBusy"/>
    <addaction name="actionBe_right_back"/>
    <addaction name="actionAway"/>
    <addaction name="actionIn_call"/>
    <addaction name="actionOut_to_lunch"/>
    <addaction name="actionAppear_off_line"/>
    <addaction name="actionSign_Out"/>
    <addaction name="separator"/>
    <addaction name="actionRegistration_info"/>
    <addaction name="actionChange_your_contact_card"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAddress_Book">
    <property name="title">
     <string>&amp;Address Book</string>
    </property>
    <addaction name="actionAdd_Contact"/>
    <addaction name="actionEdit_Contact"/>
    <addaction name="actionDelete_Contact"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Group"/>
    <addaction name="actionEdit_Group"/>
    <addaction name="actionDelete_Group"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Sphere"/>
    <addaction name="actionEdit_Sphere"/>
    <addaction name="actionDelete_Sphere"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionForum"/>
    <addaction name="actionGetting_started"/>
    <addaction name="separator"/>
    <addaction name="actionRelease_Note"/>
    <addaction name="actionCompliance"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_Log_File"/>
    <addaction name="actionDiagnostics"/>
    <addaction name="separator"/>
    <addaction name="actionAudio_Video_devices"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAddress_Book"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>33</width>
     <height>33</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTbarSignIn"/>
   <addaction name="actionTbarSignOut"/>
   <addaction name="actionTbarAddressBook"/>
   <addaction name="actionTbarSettings"/>
  </widget>
  <action name="actionOnline">
   <property name="text">
    <string>Online</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAdd_Contact">
   <property name="text">
    <string>Add Contact</string>
   </property>
  </action>
  <action name="actionEdit_Contact">
   <property name="text">
    <string>Edit Contact</string>
   </property>
  </action>
  <action name="actionDelete_Contact">
   <property name="text">
    <string>Delete Contact</string>
   </property>
  </action>
  <action name="actionAdd_Group">
   <property name="text">
    <string>Add Group</string>
   </property>
  </action>
  <action name="actionEdit_Group">
   <property name="text">
    <string>Edit Group</string>
   </property>
  </action>
  <action name="actionDelete_Group">
   <property name="text">
    <string>Delete Group</string>
   </property>
  </action>
  <action name="actionAdd_Sphere">
   <property name="text">
    <string>Add Sphere</string>
   </property>
  </action>
  <action name="actionEdit_Sphere">
   <property name="text">
    <string>Edit Sphere</string>
   </property>
  </action>
  <action name="actionDelete_Sphere">
   <property name="text">
    <string>Delete Sphere</string>
   </property>
  </action>
  <action name="actionGetting_started">
   <property name="text">
    <string>Getting started</string>
   </property>
  </action>
  <action name="actionRelease_Note">
   <property name="text">
    <string>Release Note</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
  <action name="actionCompliance">
   <property name="text">
    <string>Compliance</string>
   </property>
  </action>
  <action name="actionBusy">
   <property name="text">
    <string>Busy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionBe_right_back">
   <property name="text">
    <string>Be right back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionAway">
   <property name="text">
    <string>Away</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionIn_call">
   <property name="text">
    <string>In call</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionOut_to_lunch">
   <property name="text">
    <string>Out to lunch</string>
   </property>
  </action>
  <action name="actionAppear_off_line">
   <property name="text">
    <string>Appear off line</string>
   </property>
  </action>
  <action name="actionChange_your_contact_card">
   <property name="text">
    <string>Change your contact card</string>
   </property>
  </action>
  <action name="actionRegistration_info">
   <property name="text">
    <string>Registration info</string>
   </property>
  </action>
  <action name="actionTbarSignIn">
   <property name="text">
    <string>SignIn</string>
   </property>
  </action>
  <action name="actionTbarSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTbarAddressBook">
   <property name="text">
    <string>AddressBook</string>
   </property>
  </action>
  <action name="actionTbarSignOut">
   <property name="text">
    <string>SignOut</string>
   </property>
  </action>
  <action name="actionOpen_Log_File">
   <property name="text">
    <string>Open Log File</string>
   </property>
  </action>
  <action name="actionDiagnostics">
   <property name="text">
    <string>Diagnostics</string>
   </property>
  </action>
  <action name="actionAudio_Video_devices">
   <property name="text">
    <string>Audio/Video devices</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="doubs.qrc"/>
 </resources>
 <connections/>
</ui>
